### DISCUSSION TITLE
Kit Preview modes

### DISCUSSION THREAD

**Discussion by MotikaCraft-TheJoshua** [EssentialsX/Essentials#4608] [Ideas and Feature Suggestions] [answered]
Making Kit Preview GUI mode and possibly adding new options to "kits.yml"
-Adding toggle option like "mode:" which will have options "text" (which is default currently) and "gui" (which will preview kit in gui)
-Specifically adding ability to select what is max-slots of an inventory (so if you preview any kit, the inventory size will always be 9/18/27/...)
-Adding "kits-slots" where we can specify which slots will be used to display items/commands/money (everything that a certain kit gives to a player).
-Dummy slots for designing purpose, "dummy-slots", where we can type out all slots we want to use for dummy items.
-Dummy material, "dummy-material", where we can specify which material will be displayed in dummy slots, example: "STAINED_GLASS_PANE:15"

**Comment by JRoy** [comment, marked as answer]
Closing in favor of #3944
